house/constants.py

```diff
@@ -1,2 +1,7 @@
 HOME_PAGE_ROUTE = 'home.html'
 GLOBAL_PAGE_ROUTE = 'global/global.html'
+MINE_EXPENSES_TITLE_ROUTE = 'house_view/expenses_table.html'
+MINE_HOUSE_TABLE_ROUTE = 'house_view/house_table.html'
+MINE_HOUSE_TABLE_TITLE_ROUTE = 'house_view/house_table_title.html'
+MINE_MINE_PAGE_ROUTE = 'house_view/mine_page.html'
```

Comment:
Could you change it to just 1 Mine ? 😇

Reply:
I wanted to classify all the MINE constants that would start with the same word @LiorKGOW